airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/schemas/ads.json

```diff
@@ -0,0 +1,168 @@
+{
+  "type": ["null", "object"],
+  "properties": {
+    "advertiser_id": {
+      "type": "integer"
+    },
+    "campaign_id": {
+      "type": "integer"
```

Comment:
Are you sure that response of ads will always have `campaign_id`, `campaign_name` and other? May be better add to all fields in all schemas `null` type.

Reply:
they are required fields according to its documentation